tests/integration_tests/functional/test_cpu_template_helper.py

```diff
@@ -84,7 +84,7 @@ def fingerprint_compare(
         """Compare two fingerprint files"""
         cmd = (
             f"{self.BINARY_PATH} fingerprint compare"
-            f" --prev {prev_path} -curr {curr_path}"
+            f" --prev {prev_path} --curr {curr_path}"
```

Comment:
how has it been working before?

Reply:
Actually `test_host_fingerprint_change` still has not been run until now, because we don't have any control of microcode/BIOS updates. We're trying to find out a solution to run this test in a way not to annoy us with too many false alarms.

Reply:
Shall we consider a metric-only way of running it as well? Instead of failing the test, it would emit a high level in a metric and we can analyse that at our convenience.

Reply:
That sounds a good idea. Will include a follow-up PR.